firetweet/src/main/java/org/getlantern/firetweet/activity/support/SignInActivity.java

```diff
@@ -386,18 +413,18 @@ private void friendDefaultAccounts(final long accountId) {
             final long[] accountIds = new long[1];
             accountIds[0] = accountId;
 
-        final AsyncTwitterWrapper twitter = getTwitterWrapper();
-        twitter.createFriendshipAsync(accountId, Constants.LANTERN_ACCOUNT_ID);
-        twitter.createFriendshipAsync(accountId, Constants.FIRETWEET_ACCOUNT_ID);
-        twitter.createFriendshipAsync(accountId, Constants.MANOTO_TV_ACCOUNT_ID);
-        twitter.createFriendshipAsync(accountId, Constants.MANOTO_NEWS_ACCOUNT_ID); 
+            final AsyncTwitterWrapper twitter = getTwitterWrapper();
+            twitter.createFriendshipAsync(accountId, Constants.LANTERN_ACCOUNT_ID);
+            twitter.createFriendshipAsync(accountId, Constants.FIRETWEET_ACCOUNT_ID);
+            twitter.createFriendshipAsync(accountId, Constants.MANOTO_TV_ACCOUNT_ID);
+            twitter.createFriendshipAsync(accountId, Constants.MANOTO_NEWS_ACCOUNT_ID); 
 
             String initialTweetText = this.getString(R.string.initial_tweet);
 
             twitter.updateStatusAsync(accountIds, initialTweetText, null, null, -1,
                     false);
 
-            mPreferences.edit().putBoolean("firstRun", true).apply();
+            mPreferences.edit().putBoolean("firstRun", true).commit();
```

Comment:
Same as above. Moreover, `FIRST_RUN` might be ambiguous here, as when it's true, it already run for the first time. Something like `ALREADY_RUN` (I'm not good at English naming) ?